components/calendar/src/lib.rs

```diff
@@ -2,14 +2,19 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
+#![cfg_attr(not(any(test, feature = "std")), no_std)]
+
```

Comment:
If we're promoting this crate from experimental to components… can you add module-level docs, or at least make sure there is a ticket blocking 0.4 to track that?  We should keep the components directory "ready to release" as much as possible.

Reply:
Suggestion: Similarly, this needs `#![warn(missing_docs)]` enabled for it, with at least suppressions for missing docs and a follow-up bug.

Reply:
This is part of my next steps.

Reply:
(I'm trying to keep this PR to be restricted to code moving)